apps/aws-amplify/src/locations/Sidebar.tsx

```diff
@@ -50,9 +52,10 @@ const Sidebar = () => {
         isFullWidth={true}
         isLoading={isLoading}
         isDisabled={isLoading}
-        onClick={buildAmplifyApp}>
-        {isLoading ? 'Building' : 'Build website'}
+        onClick={handleBuildAppActionCall}>
+        {isLoading ? 'Initiating build...' : 'Build now'}
       </Button>
+      <span>Last build started: {lastBuildInitiated}</span>
```

Comment:
Also, there could probably be some more padding between the button at the `Last build started` text.